frontend/packages/jobs/src/functions/processGenerateDocsSuggestion.ts

```diff
@@ -52,7 +52,15 @@ export async function processGenerateDocsSuggestion(payload: {
       return {
         id: filename,
         title: filename,
-        content: fileData.content || '',
+        content: fileData.content
+          ? Buffer.from(fileData.content, 'base64')
+              .toString('utf-8')
+              .replace(/\\/g, '\\\\')
+              .replace(/\n/g, '\\n')
+              .replace(/\r/g, '\\r')
+              .replace(/\t/g, '\\t')
+              .replace(/"/g, '\\"')
+          : '',
```

Comment:
https://github.com/liam-hq/liam/blob/258096a081e2da84dafec4a8dee583a6dc1c5907/frontend/packages/jobs/src/functions/processGenerateDocsSuggestion.ts#L67-L71 

I don't really understand why JSON.stringify(docsArray) is called here, but a process similar to JSON.stringify is required for each individual string before that.

Perhaps it would be better to use JSON.stringify(docsArray.join("\n")) instead of JSON.stringify(docsArray)?

Reply:
You are right...as for docsArray, I think it is the same before and after.
I think it is fine as it is because `docsArray.join(“\n”) `would break the json structure, but `JSON.stringify(docsArray) `doesn't seem necessary.

Reply:
Thank you for your confirmation!